src/util/psbt/map/output.rs

```diff
@@ -55,6 +73,41 @@ pub struct Output {
     pub unknown: BTreeMap<raw::Key, Vec<u8>>,
 }
 
+/// Taproot Tree representing a finalized [`TaprootBuilder`]
```

Comment:
Not sure why we need this type here and not in taproot mod

Reply:
Because `Builder` is allowed to be an unfinalized state in taproot mod. The complete finalized state is only required when encoding/decoding psbts. 

If there is any use of a finalized builder outside psbts, we can move it there. I don't have a strong opinion here, I am willing to move to `taproot.rs` if you feel it makes sense there.

Reply:
No, I do not think this way, it was just a question